encord/project.py

```diff
@@ -643,8 +643,8 @@ def model_inference(
 
     def model_train(
         self,
-        uid: str,
-        label_rows: Optional[List[str]] = None,
+        uid: UUID,
```

Comment:
This is a public endpoint so it's better to convert it to a  `str | UUID`, then converted to uuid when we need